conf/evolutions/default/9.sql

```diff
@@ -0,0 +1,46 @@
+# core model types
+# --- !Ups
+
+DROP TABLE interest;
+DROP SEQUENCE interest_id_seq;
+
+CREATE SEQUENCE interest_id_seq;
+CREATE TABLE interest (
+  id integer NOT NULL DEFAULT nextval('interest_id_seq'),
```

Comment:
Yes, good point. I had started the DDL to be DB-agnostic, but there might already be some Postgresqlisms that have crept in. So why not go whole-hog?